### ISSUE TITLE
Quantified Dynamics-Property Relationships: Data-Efficient Protein Engineering with Machine Learning of Protein Dynamics

### ISSUE THREAD

**Issue by agitter** [gitter-lab/low-n-mlde-review#3]
https://doi.org/10.1101/2025.04.23.650227

> Machine learning has proven to be very powerful for predicting mutation effects in proteins, but typically only when a large amount of training data is available. Because experiments to collect training data are often expensive, time-consuming, and/or otherwise limited, alternatives that make good use of small amounts of data to guide protein engineering are of high potential value. One potential alternative to large-scale benchtop experiments for collecting training data is high-throughput molecular dynamics simulation; however, to date this source of data has been largely absent from the literature. Here, I introduce a new method for selecting desirable protein variants based on quantified relationships between a small number of experimentally determined labels and descriptors of their dynamic properties. These descriptors are provided by deep neural networks trained on data from molecular dynamics simulations of variants of the protein of interest. I demonstrate that this approach can obtain very highly optimized variants based on small amounts of experimental data, greatly outperforming machine learning-guided directed evolution with the same amount of experimental data. Furthermore, I show that quantified dynamics-property relationships based on only a handful of experimentally labeled example sequences can be used to accurately predict the key residues that are most relevant to determining the property in question, even when that information could not have been known or predicted based on either the molecular dynamics simulations or the experimental data alone. This work establishes a new, general, and highly practical tool for incorporating protein dynamics information to guide data-efficient protein engineering.

**Comment by agitter** [comment]
The journal version is out: https://doi.org/10.1021/acs.jcim.5c01813
